kubetest2-gke/deployer/deployer.go

```diff
@@ -65,11 +65,17 @@ var (
 	// m[3]: unique hash (used as nonce for firewall rules)
 	poolRe = regexp.MustCompile(`zones/([^/]+)/instanceGroupManagers/(gk[e|3]-.*-([0-9a-f]{8})-grp)$`)
 
-	urlRe           = regexp.MustCompile(`https://.*/`)
+	urlRe = regexp.MustCompile(`https://.*/`)
```

Comment:
@amwat are we not linting gofmt in CI?

Reply:
we should be https://github.com/kubernetes-sigs/kubetest2/blob/master/hack/tools/.golangci.yml#L21

Reply:
Oh, we do gofmt. I added an empty line between regexps and the default node pools. that's why it got reformatted.

Reply:
ah, thanks :-)